src/modules/_[spaceId]/HomePage/HomePage.tsx

```diff
@@ -88,50 +121,94 @@ export default function HomePage({
     : [launchCommunityButton, integrateChatButton]
 
   return (
-    <DefaultLayout>
+    <DefaultLayout
+      navbarProps={{
+        customContent: (logo, auth, colorModeToggler) => {
+          return (
+            <HomePageNavbar
+              search={search}
+              setSearch={setSearch}
+              auth={auth}
+              colorModeToggler={colorModeToggler}
+              logo={logo}
+              setFocusedElementIndex={setFocusedElementIndex}
+              onUpClick={onUpClick}
+              onDownClick={onDownClick}
```

Comment:
It looks dirty for me, lot of props for useStates; maybe better use Context for the component (Or zustand)

Or for first iteration, just collect it to one object and send as `searchProps`

Reply:
moved props into an object searchProps